app/models/user.rb

```diff
@@ -0,0 +1,14 @@
+class User < ApplicationRecord
+  has_many :events, foreign_key: :creator_id
+  has_many :invitations, foreign_key: :attendee_id
+  has_many :attended_events, through: :invitations, source: :event
+  validates :email, :name, uniqueness: true, presence: true
```

Comment:
- [ ] The email validation can be better, try using regular expressions to make sure the value entered is a valid email or one of Rails inbuild email validators. Read more in [this blog post](https://blog.mailtrap.io/rails-email-validation/).

- [ ] Validate the name of the user too so it's at least not nill.